aten/src/ATen/native/native_functions.yaml

```diff
@@ -4654,22 +4654,21 @@
 
 - func: prelu(Tensor self, Tensor weight) -> Tensor
   variants: function, method
+  autogen: prelu.out
```

Comment:
Do you actually need a prelu.out since this is CompositeImplicit ?

Reply:
Leaving it as people may still want to call `torch.prelu(x, out=out)`